src/main/java/edu/kis/powp/jobs2d/drivers/DriverChangeObserver.java

```diff
@@ -1,25 +1,21 @@
 package edu.kis.powp.jobs2d.drivers;
 
-import edu.kis.powp.jobs2d.Job2dDriver;
 import edu.kis.powp.jobs2d.enums.RECORDING_STATUS;
-import edu.kis.powp.jobs2d.features.DriverFeature;
 import edu.kis.powp.jobs2d.features.RecordingFeature;
 import edu.kis.powp.observer.Subscriber;
 
```

Comment:
You should find a better name for DriverChangeObserver (such a name suits literally every driver change observer). It doesn't tell anything about its purpose.